src/components/Settings.js

```diff
@@ -1,5 +1,63 @@
-import React from 'react';
+import React, { useContext } from 'react';
 
-const Settings = () => <h1>Settings</h1>;
+import ReactSlider from 'react-slider';
+
+import { LocationsContext } from '../context/LocationsContext';
+import { MapCenterContext } from '../context/MapCenterContext';
+
+const Settings = () => {
+  const { wait, setWait } = useContext(LocationsContext);
+  const { userLocationShared } = useContext(MapCenterContext);
+
+  const handleChange = (value) => {
+    setWait(value);
+  };
+
+  return (
+    <>
+      <div>
+        <h1>Settings </h1>
+        <h3 className="settings-subheader">
```

Comment:
Quetioning the user of an `<h3>` here. Proper semantics would call for an `<h2>` to be the next header after and `<h1>` but in this case I would think a simple `<p>` would suffice, no?

Reply:
Good point, the ``<h2>`` tag has been replaced